Common/include/option_structure.hpp

```diff
@@ -1281,11 +1285,29 @@ inline LM_ParsedOptions ParseLMOptions(const LM_OPTIONS *LM_Options, unsigned sh
  */
 enum class SPECIES_MODEL {
   NONE,              /*!< \brief No scalar transport model. */
-  SPECIES_TRANSPORT,    /*!< \brief Passive scalar transport model. */
+  SPECIES_TRANSPORT,    /*!< \brief species transport model. */
+  FLAMELET,          /*!< \brief flamelet model. */
 };
 static const MapType<std::string, SPECIES_MODEL> Species_Model_Map = {
   MakePair("NONE", SPECIES_MODEL::NONE)
   MakePair("SPECIES_TRANSPORT", SPECIES_MODEL::SPECIES_TRANSPORT)
+  MakePair("FLAMELET", SPECIES_MODEL::FLAMELET)
+};
+
+/* the order matters: */
+/*!
+* \brief progress variable and enthalpy are the first and second entry in the lookup table
+*/
```

Comment:
Most of our comments are written with capitals, please try to be consistent we have enough styles already :)
```suggestion
/*!
 * \brief Progress variable and enthalpy are the first and second entries in the lookup table.
 * \note The order matters.
 */
```